test/unit/vl-mapactions-box-select-action.test.js

```diff
@@ -58,7 +59,13 @@ describe('box select action', () => {
   it('zal bij het slepen van de box selectie, de features van de layer toevoegen aan de hover interactie', () => {
     const VlboxSelectAction = createVlBoxSelectAction();
     VlboxSelectAction.dragBoxInteraction.dispatchEvent('boxdrag');
-    expect(VlboxSelectAction.hoverInteraction.getFeatures().getArray()).contain(intersectingFeature);
+    expect(VlboxSelectAction.hoverInteraction.getFeatures().getArray()).to.have.members([feature1, feature2]);
```

Comment:
Use camelCase for constants, `boxSelectAction` bv

Reply:
Yes, ge hebt gelijk. Copy pasta en zelf niet opgemerkt, maar heb het dus gefixed :)